### PR TITLE
EOS-24431: Containerization Support for Storage PVCs

### PR DESCRIPTION
**Pull request by jjxsg**
# Problem Statement
Containerization Support for Storage PVCs

# Design
Sample PV/PVC configuration for shared storage, logs and config. 

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent.

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: test/volumes/cortx-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
```
Comment (ujjwalpl) [inline test/volumes/cortx-deployment.yml:1]: Please move this code to cortx-prvsnr/experiments folder.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: test/volumes/cortx-pv-shared.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name: cortx-pv-shared
+spec:
+  storageClassName: shared-pv
+  capacity:
+    storage: 20Gi
+  accessModes:
+    - ReadWriteMany
+  volumeMode: Filesystem
+  hostPath:
+    path: /data/pv/shared
```
Comment (cdeshmukh) [inline test/volumes/cortx-pv-shared.yml:13]: this path needs to be replaced from config dynamically, before PV/PVC can be deployed

File: test/volumes/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -x
+
+SCRIPT_DIR=$(dirname $0)
+
+kubectl create namespace cortx
+
+kubectl apply -f "$SCRIPT_DIR"/cortx-pv-logs.yml --namespace cortx
```
Comment (cdeshmukh) [inline test/volumes/deploy.sh:7]: namespace we can have it as seagate-cortx, as a recommendation ofcourse

**Comment by jjxsg** [comment]
Closing this PR as PV/PVC infrastructure code is already in place. Finding of the POC will be updated in the confluence page.

**Pull request closed without merging**